src/Client/Updater.php

```diff
@@ -535,9 +546,8 @@ private function fetchFile(string $fileName, int $maxBytes = self::MAXIMUM_DOWNL
      */
     public function download(string $target, ...$extra): PromiseInterface
     {
-        if (!$this->isRefreshed) {
-            $this->refresh();
-        }
+        $this->refresh();
```

Comment:
I don't think we need this change be change because `refresh()` method will take care of returning early since we are not passing `$force`

Reply:
NVB I had the  +/- reversed in my head. 🤦🏼‍♂️